src/Illuminate/Collections/Arr.php

```diff
@@ -420,7 +418,7 @@ public static function isAssoc(array $array)
      */
     public static function isList($array)
```

Comment:
@taylorotwell do we really need to keep maintaining this method? Maybe mark it as deprecated and remove in next major release?

Reply:
@antonkomarev don't think it's worth it but feel free to attempt a pr

Reply:
@driesvints @antonkomarev I think they are needed for package developers since if you are writing a package you do not know which laravel and PHP version the package will be installed on.
So we can not deprecate and remove them.